src/main/java/com/minecolonies/coremod/colony/colonyEvents/raidEvents/pirateEvent/ShipSize.java

```diff
@@ -1,42 +1,33 @@
 package com.minecolonies.coremod.colony.colonyEvents.raidEvents.pirateEvent;
 
 import com.minecolonies.api.util.constant.ColonyConstants;
-import net.minecraft.entity.EntityType;
-
-import static com.minecolonies.api.entity.ModEntities.*;
 
 /**
  * Enum for ship sizes.
  */
 public enum ShipSize
 {
-    SMALL(ColonyConstants.SMALL_HORDE_SIZE, ColonyConstants.SMALL_PIRATE_SHIP, ColonyConstants.SMALL_HORDE_MESSAGE_ID, 1, PIRATE, PIRATE),
+    SMALL(ColonyConstants.SMALL_HORDE_SIZE, ColonyConstants.SMALL_SHIP, ColonyConstants.SMALL_HORDE_MESSAGE_ID, 2, 2, 0, 0),
     MEDIUM(ColonyConstants.MEDIUM_HORDE_SIZE,
-      ColonyConstants.MEDIUM_PIRATE_SHIP,
+      ColonyConstants.MEDIUM_SHIP,
       ColonyConstants.MEDIUM_HORDE_MESSAGE_ID,
-      3,
-      PIRATE,
-      ARCHERPIRATE,
-      CHIEFPIRATE),
+      4,
```

Comment:
medium ship now has 4 spawners?

Reply:
No, 6, good point

Reply:
6? It was 3 originally :D

Reply:
Gotta double check then, Carlsanor switched those things up =D